doc/7_java_api.md

```diff
@@ -138,6 +138,30 @@ is also one overloaded `mapRowTo` methods which allows to specify a custom `Colu
 mapper can be found in [Working with user-defined case classes and tuples](4_mapper.md) and
 [Customizing the mapping between Scala and Cassandra](6_advanced_mapper.md).
 
+Since 1.2, it is possible to easily provide custom column name to property name translation by
+`select` method.
+
+Example:
+Say we have a table `people2` with columns `id INT`, `last_name TEXT`, `date_of_birth TIMESTAMP` and
+we want to map the rows of this table to objects of `Person` class.
+
+```java
+CassandraJavaRDD<Person> rdd = javaFunctions(sc).cassandraTable("ks", "people2", mapRowTo(Person.class)).select(
+        plain("id"),
```

Comment:
Why is it called "plain"? Little surprising to me. Wouldn't it be more readable to call it "column" ?